### ISSUE TITLE
jenkins-x-gc-previews job do nothing after PR is merged (bitbucket).

### ISSUE THREAD

**Issue by mrpity** [jenkins-x/jx#1312]
### Summary 
jenkins-x-gc-previews job do nothing after PR is merged/declined on Bitbucket 
### Steps to reproduce the behavior
```
- create preview env
- close PR
```
### Jx version

The output of `jx version` is:
 1.3.100

```
NAME               VERSION
jx                 1.3.100
jenkins x platform 0.0.1827
kubernetes cluster v1.10.5
kubectl            v1.11.0
helm client        v2.10.0-rc.1+gaa98e7e
helm server        v2.10.0-rc.1+gaa98e7e
git                git version 2.11.0
```

### Kubernetes cluster

What kind of Kubernetes cluster are you using & how did you create it?
- kubernetes on-premise
- kubespray
### Operating system / Environment
Ubuntu 16.04.4 LTS
### Expected behavior
I expect that all preview envs should be deleted with help of jenkins-x-gc-previews job, when
PR is merged/declined. 
### Actual behavior
- jenkins-x-gc-previews job do nothing 
- manual run of command ('jx gc previews') also do nothing
```
# jx gc previews
# kubectl get pods | grep prev
jenkins-x-gc-previews-1531882800-5bh8p           0/1       Completed   0          7h
jenkins-x-gc-previews-1531893600-9tssx           0/1       Completed   0          4h
jenkins-x-gc-previews-1531904400-rvzb5           0/1       Completed   0          1h
# jx get previews
PULL REQUEST                                              NAMESPACE                            APPLICATION
https://bitbucket.org/whirlsoftware/wsec_frontend/pull/12 jx-whirlsoftware-wsec-frontend-pr-12 http://fe.jx-whirlsoftware-wsec-frontend-pr-12.k8s.whirl.sg
https://bitbucket.org/whirlsoftware/wsec_frontend/pull/14 jx-whirlsoftware-wsec-frontend-pr-14 http://fe.jx-whirlsoftware-wsec-frontend-pr-14.k8s.whirl.sg

```

**Comment by mrpity** [comment]
AS I know it should be fixed here:
https://github.com/jenkins-x/jx/pull/1310

**Comment by rawlingsj** [comment]
Thanks - yes that PR has been merged and we're awaiting the release to be available.  We're being rate limited by GitHub a little at the moment so we may need to wait an hour until the new version is ready.

**Comment by rawlingsj** [comment]
This was confirmed as working on Slack earlier with the fix from #1310
